<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu_window</class>
 <widget class="QDialog" name="menu_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="admin_login_button">
     <property name="text">
      <string>i'm admin</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="user_login_button">
     <property name="text">
      <string>i'm user</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="exit_button">
     <property name="text">
      <string>i'm exit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
